docs/Examples.md

```diff
@@ -229,19 +215,24 @@ spec:
 EOF
 ```
 
-In the above environment, cluster master controls the [applications loaded](#installing-splunk-apps) to all
-the parts of the indexer cluster, and the indexer services that it creates select the indexers
```

Comment:
@kashok-splunk Any idea what this means? Other than the apps part, I find the rest quite confusing.

Reply:
Looks like this is for apps installation that cluster master installs all the applications on the indexers

Reply:
"and through the indexer services created by the cm, the cm has the knowledge about the indexers" this is what I get

Reply:
Thank you! That helps a lot.